force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        if (a == null || b == null) {
+        return null;
+        }
+        return a+b;
```

Comment:
Great work. Just add spacing around the operator for readability here: `return a + b;`